github/compliance/issue.py

```diff
@@ -192,42 +164,39 @@ def _update_issue(
 
 
 def create_or_update_issue(
-    component: cm.Component,
-    artifact: cm.Artifact,
+    labels: frozenset[str],
     repository: github3.repos.Repository,
-    body:str,
+    body: str,
     known_issues: typing.Iterable[github3.issues.issue.ShortIssue],
-    title:str=None,
+    title: str,
     assignees: typing.Iterable[str]=(),
     milestone: github3.issues.milestone.Milestone=None,
-    issue_type: str=_label_bdba,
     extra_labels: typing.Iterable[str]=None,
```

Comment:
why did you remove extra_labels from the other functions, but keep it here?

Reply:
The other func retrieved repo labels based off component+artifact and the passed extra labels.
As we now pass in labels directly, I merged those to only pass one `labels` arg.

This func however has a special semantic for extra labels, that is considering the passed `preserve_labels_regexes`. Therefore I kept the separation here.